packages/ui/ToggleItem/ToggleItem.module.less

```diff
@@ -1,3 +1,7 @@
 .toggleItem {
 	/* Placeholder for CSS overrides */
+	.itemIcon {
+		background-position: 0;
```

Comment:
I do not believe it is wise to arbitrarily reassign these rules to different values, and _especially_ not in `ui/ToggleItem`. These are properties that Moonstone has defined and all icons adhere to. If an app developer wants them to be different for whatever reason, it is advised that they do this in their own app on their one component.

Reply:
I agree with your opinion. I think I missed that point! So, I've removed this change.